### DISCUSSION TITLE
MTP support to mount Android devices on MacOS

### DISCUSSION THREAD

**Discussion by megseoh** [iterate-ch/cyberduck#14615] [Ideas]
I would love to be able to easily mount my Android device on my Mac – to easily transfer files between them.
Thanks for considering it!

**Comment by AliveDevil** [comment]
Build and support libmtp for Windows x64 (arm64 in future), macOS X64, macOS ARM64, Linux armhf, arm64 & x64.
Then build native wrapper library around [libmtp](https://github.com/libmtp/libmtp) in Java.

This needs to distribute [libusb](https://github.com/libusb/libusb) as well, which may come with its own [limitations](https://github.com/libusb/libusb/wiki/FAQ#running-libusb).

If someone's up to the task, contributions are welcome.
